workplace/Living in Open Workspaces.md

```diff
@@ -0,0 +1,80 @@
+## Living in Open Workspaces
+
+Total time: 35 minutes
+
+- Lesson: 15 min
+- Check for Understanding: 20 min
+
+## Motivation
+
+In a [recent study](https://www.business2community.com/human-resources/how-to-design-a-workspace-that-boosts-employee-motivation-02222288), research company IPSOS, discovered that a workspace focused on employee happiness is a powerful driver for productivity.
+
+The IPSOS study showcases the urgent need for more engaging workspaces:
+
+- 85% of workers feel they don’t work in a space where they can share ideas
+- 87% feel there isn’t a place to work in teams
+- 84% don’t feel a sense of belonging or a willingness to be a part of company culture
+
+## Objectives
+
+Participants will be able to:
+
+- Know in detail about workspace management
+- Manage themselves properly in work environment
+- Learn the skills to be a leader in the workplace
+
+## Materials
+
+- [Open office environment and ground rules](https://www.coworkingresources.org/blog/open-office-etiquette-and-ground-rules)- It lists the regulations which can be followed to set up a positive workspace.
+- [Managing Workplace Conflict](https://www.shrm.org/resourcesandtools/tools-and-samples/toolkits/pages/managingworkplaceconflict.aspx)- This toolkit provides an overview about how employers and employees can manage and resolve workplace conflict
+
+
+## Lesson
+
```

Comment:
Can you just tell them to read your Workplace Rules link, and the other materials links, and tell them what parts to read, what to notice, and how long to spend on it?

I'm not convinced that any content under Lesson or Common Mistakes & Misconceptions is original.

Reply:
I think adding only the links would be useful.

Reply:
Ok, please remove this content and tell them which section of the link to read, and how long it will take.

Reply:
Ok, please remove this content and tell them which section of the link to read, and how long it will take.